### ISSUE TITLE
eqxwebcs type parameter error on clean generate

### ISSUE THREAD

**Issue by CumpsD** [jet/dotnet-templates#44]
```bash
dotnet new eqxwebcs -ag -t -e
```

In `ServiceBuilder`:

```csharp
        public Todo.Service CreateTodoService() =>
            new Todo.Service(
                _handlerLog,
                _context.Resolve(
                    EquinoxCodec.Create(Todo.Event.Encode, Todo.Event.TryDecode),
                    Todo.State.Fold,
                    Todo.State.Initial,
                    Todo.State.IsOrigin,
                    Todo.State.Snapshot));

        public Aggregate.Service CreateAggregateService() =>
            new Aggregate.Service(
                _handlerLog,
                _context.Resolve(
                    EquinoxCodec.Create(Aggregate.Event.Encode, Aggregate.Event.TryDecode),
                    Aggregate.State.Fold,
                    Aggregate.State.Initial,
                    Aggregate.State.IsOrigin,
                    Aggregate.State.Snapshot));
```

![image](https://user-images.githubusercontent.com/1201644/73958421-32dafd80-4908-11ea-91f5-e1a8af7239d3.png)

**Comment by bartelink** [comment]
On master but not released; ultimate solution for this is #2, but that's not a 2m job

**Comment by bartelink** [comment]
Fixed in 3.10.0 (which is on the way to the nuget)
